src/Repository/RepublicanSilenceRepository.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace AppBundle\Repository;
+
+use AppBundle\Entity\RepublicanSilence;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Symfony\Bridge\Doctrine\RegistryInterface;
+
+class RepublicanSilenceRepository extends ServiceEntityRepository
+{
+    public function __construct(RegistryInterface $registry)
+    {
+        parent::__construct($registry, RepublicanSilence::class);
+    }
+
+    public function findStarted(\DateTimeInterface $startDate): iterable
```

Comment:
Maybe add a doc block to say it's an array of RepublicanSilence ? RepublicanSilence[]